docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -0,0 +1,25 @@
+# New Language Feature Check Integration Process [WIP]
+
+## Check Update Procedure
+
+### Consider Similar Tokens
+### Java Enhancement Proposals (JEPS)
+### Always Impacted Checks
+
+- illegalToken
+- illegalTokenText 
+- whitespace
+- indentation
+### Real Usages Examples in Large Codebases
+### Review Other Static Analysis Tools
```

Comment:
Not exactly for this line, but let's leave out the openjdk mailing list for now, and create a separate issue for this.

Reply:
@mahfouz72 ping, please open an issue for this

Reply:
https://github.com/checkstyle/checkstyle/issues/14922